kolibri/core/device/test/test_api.py

```diff
@@ -462,3 +467,52 @@ def test_device_name_max_length(self):
                 exceeds_max_length_name,
                 format="json",
             )
+
+
+class UserSyncStatusTestCase(APITestCase):
+    @classmethod
+    def setUpTestData(cls):
+        provision_device()
+        cls.facility = FacilityFactory.create()
+        cls.superuser = create_superuser(cls.facility)
+        cls.user1 = FacilityUserFactory.create(facility=cls.facility)
+        cls.classroom = ClassroomFactory.create(parent=cls.facility)
+        cls.classroom.add_member(cls.user1)
+        cls.classroom.add_coach(cls.superuser)
+        syncdata = {
+            "id": uuid.uuid4().hex,
+            "start_timestamp": timezone.now(),
+            "last_activity_timestamp": timezone.now(),
+            "active": False,
+            "is_server": False,
+            "client_instance": True,
+            "server_instance": False,
+            "extra_fields": {},
+        }
+        syncsession = SyncSession.objects.create(**syncdata)
+        data = {
+            "user_id": cls.user1.id,
+            "sync_session": syncsession,
+            "queued": True,
+        }
+        UserSyncStatus.objects.create(**data)
+
+    def setUp(self):
+        self.client.login(
+            username=self.superuser.username,
+            password=DUMMY_PASSWORD,
+            facility=self.facility,
+        )
+
+    def test_usersyncstatus_list(self):
+        response = self.client.get(reverse("kolibri:core:usersyncstatus-list"))
+        expected_count = UserSyncStatus.objects.count()
+        self.assertEqual(len(response.data), expected_count)
+
+    def test_user_sync_status_class_single_user_for_filter(self):
+        response = self.client.get(
+            reverse("kolibri:core:usersyncstatus-list"),
+            data={"user_id": self.user1.id},
```

Comment:
Here you are filtering by `user_id` but the `FilterSet` for this endpoint specifies `user` as the parameter.

You should create another UserSyncStatus during your test setup for this test to be meaningful.